src/include/machnet_engine.h

```diff
@@ -241,6 +241,19 @@ class MachnetEngineSharedState {
     return true;
   }
 
+  int GetQueueId(const net::Ipv4::Address &ipv4_addr,
+                 const net::Udp::Port &port) {
+    auto it = listeners_to_rxq.find({ipv4_addr, port});
+    if (it == listeners_to_rxq.end()) {
+      return -1;
+    }
+
+    int res = it->second;
+    LOG(WARNING) << "found the value ########### " << res;
```

Comment:
This is debugging code and should be omitted. We shall use WARNING messages only when it's critical to print something. This is not critical; this function might return `std::nullopt` because there is simply no listener.

You can add a more descriptive `INFO` message if needed: 

`LOG(INFO) << "No listener registered on " << ipv4_addr.to_string() << ":" << port.to_string();`